www-apps/gitea/files/gitea.service-r2

```diff
@@ -0,0 +1,28 @@
+[Unit]
+Description=Gitea service
+Documentation=https://docs.gitea.io/
+
+AssertPathIsDirectory=/var/lib/gitea
+AssertPathIsReadWrite=/var/lib/gitea
+
+After=network.target
+Requires=network.target
+After=mysqld.service
+After=postgresql.service
```

Comment:
Please see https://bugs.gentoo.org/671596. Seems like the postgresql versions need to be included here.